src/Match.js

```diff
@@ -110,6 +114,25 @@ export default class Match extends EventEmitter {
     return `${this.homeTeam.getName()} / ${this.awayTeam.getName()}`;
   }
 
+  getTeam(teamId) {
+    return (
+      find([this.homeTeam, this.awayTeam], team => team.getId() === teamId) ||
+      null
+    );
+  }
+
+  getPlayer(playerId) {
+    const team = find([this.homeTeam, this.awayTeam], team =>
```

Comment:
on peut faire un reduce

`([this.homeTeam, this.awayTeam]).reduce(acc,team)=> acc ? acc : acc = team.getPlayer(playerId), null) || null`

Reply:
Ça y est, tu es le pro du `reduce` 😄

Reply:
Done ✅